lte/protos/subscriberdb.proto

```diff
@@ -213,25 +219,62 @@ message SubscribersDigest {
   string md5_base64_digest = 1;
 }
 
+message CheckSubscribersInSyncRequest {
+  // flat_digest is the deterministic digest of the full set of subscriber
+  // objects stored on the client side.
+  SubscribersDigest flat_digest = 1;
```

Comment:
SubscribersDigest: rename `SubscribersDigest` to just `Digest`, since we're using it both for single subs and for groups of subs